lib/defaults/defaults.go

```diff
@@ -368,9 +368,9 @@ const (
 	LimiterPasswordlessPeriod = 1 * time.Minute
 	// LimiterPasswordlessAverage is the default average for passwordless
 	// limiters.
-	LimiterPasswordlessAverage = 10
+	LimiterPasswordlessAverage = 20
```

Comment:
These values are no longer specific to passwordless interactions, are they?

Reply:
AFAIK they are only used in passwordless interactions, but there may be something I am missing.

Reply:
Looking at the code, I think Zac is right, so the `Passwordless` part should be removed.